README.md

```diff
@@ -60,7 +60,8 @@ Another unique advantage of RP2 is [transparent computation](https://github.com/
 RP2 currently supports the following accounting methods:
 * US: [FIFO](https://www.investopedia.com/terms/f/fifo.asp), [LIFO](https://www.investopedia.com/terms/l/lifo.asp) and [HIFO](https://www.investopedia.com/terms/h/hifo.asp). Note that these methods use universal application (not per-wallet application), as explained [here](https://www.forbes.com/sites/shehanchandrasekera/2020/09/17/what-crypto-taxpayers-need-to-know-about-fifo-lifo-hifo-specific-id/);
 * Spain: FIFO;
-* Japan: Total Average Method.
+* Japan: Total Average Method;
+* Ireland: FIFO. In Ireland specific rules apply to assets bought and sold within a 4 week window. See [here](https://www.revenue.ie/en/tax-professionals/tdm/income-tax-capital-gains-tax-corporation-tax/part-19/19-04-03.pdf) and [here](https://www.revenue.ie/en/gains-gifts-and-inheritance/transfering-an-asset/selling-or-disposing-of-shares.aspx). Note that this is **NOT** catered for. 
```

Comment:
Interesting: perhaps this could be handled (in a separate PR) with a new Ireland-specific accounting method plugin, a variation of FIFO that treats specially lot fractions that are sold within 4 weeks of the last purchase.